Sources/MongoSwift/Operations/ListDatabasesOperation.swift

```diff
@@ -0,0 +1,91 @@
+import mongoc
+
+/// Options to use when listing available databases.
+public struct ListDatabasesOptions: Encodable {
```

Comment:
since users can't actually pass this in anywhere, should it be internal?

Reply:
I think you could also keep `filter` in this struct then, and it would simplify your encoding logic in the operation below.

Reply:
Done and done